usaspending_api/references/views.py

```diff
@@ -29,15 +29,17 @@ def post(self, request, geocomplete=False, format=None):
         return Response(response_data)
 
 
-class AgencyAutocomplete(APIView):
+class AgencyAutocomplete(FilterQuerysetMixin,
```

Comment:
Thanks for converting to the new views/mixins! That should allow us to hook in to the DRF caching stuff, too.